src/styles/style.css

```diff
@@ -0,0 +1,7 @@
+/* add styles here */
+
+@import 'blocks/stars.css';
```

Comment:
Avoid using '@import' to include CSS files, as it can lead to performance issues due to its synchronous loading behavior. Instead, consider linking to separate CSS files in your HTML using multiple <link> tags, which allows for parallel loading.